rotkehlchen/api/v1/schemas.py

```diff
@@ -3694,3 +3694,30 @@ def make_calendar_query(
             identifiers=data['identifiers'],
         )
         return {'filter_query': filter_query}
+
+
+class CalendarReminderCommonEntrySchema(Schema):
+    event_id = fields.Integer(
+        required=True,
+        validate=webargs.validate.Range(min=1, error='event_id has to be bigger than 0'),
```

Comment:
Why do you need this check? I know sqlite rowid has to start from 1 but no need to check this here. This is just an id the frontend will give us. We give it to them anyway. No need for a check here. If they give us a non-existing id it will be seen by the backend anyway later and handled properly.